buildlib/tools/builds.sh

```diff
@@ -201,31 +201,33 @@ build_ugni() {
 # Build CUDA
 #
 build_cuda() {
-	if az_module_load $CUDA_MODULE
-	then
-		if az_module_load $GDRCOPY_MODULE
+	if [[ ! $CONTAINER == *"rocm"* ]]; then
```

Comment:
1. can we check something related to Cuda? it's better that each build func will check its own requirements and not that the "other" requirements are present.
2. better exit the function if the condition is not met, to reduce indentation level

Reply:
Sounds reasonable. However, there is no Cuda preinstalled on those images:
https://github.com/openucx/ucx/blob/master/buildlib/dockers/centos7.Dockerfile#L5-L31